test/lib/price-feed.test.ts

```diff
@@ -0,0 +1,25 @@
+import chai from 'chai';
+import {CoinbaseFeed} from '../../lib/feed';
+import nock from 'nock';
+
+const {expect} = chai;
+
+global.fetch = require('node-fetch');
+
+describe('Price Feed', () => {
+  it('should fetch coinbase spot price', async () => {
+    nock('https://api.coinbase.com')
+      .get('/v2/prices/spot?currency=GBP')
+      .reply(200, {
+        data: {
+          base: 'BTC',
+          currency: 'GBP',
+          amount: '8893.89058866'
+        }
+      });
+
+    const feed = new CoinbaseFeed('GBP');
+    const amount = await feed.getSpotPrice();
+    expect(amount).to.eq(8893.89058866);
+  });
+});
```

Comment:
We should add a test here that checks if we get a "200" and the right "base" and "currency" response for the most common currencies we want to show in the UI. Should be USD, EUR, and GBP for now.

Reply:
The difficulty here is that we don't want to end up testing the coinbase api, rather our query code which is why we use the `nock` package to mock the responses.

Reply:
I in terms of unit testing I think it's enough with what we have here. Testing what actually comes back from the API would be a better fit for integration testing.